enterprise/api_client/enterprise_catalog.py

```diff
@@ -151,3 +152,15 @@ def contains_content_items(self, catalog_uuid, content_ids):
         query_params = {'course_run_ids': content_ids}
         endpoint = getattr(self.client, self.ENTERPRISE_CATALOG_ENDPOINT)(catalog_uuid)
         return endpoint.contains_content_items.get(**query_params)['contains_content_items']
+
+    @JwtLmsApiClient.refresh_token
+    def enterprise_contains_content_items(self, enterprise_uuid, content_ids):
+        """
+        Checks whether an enterprise customer has any catalogs that contain the provided content ids.
+
+        The endpoint does not differentiate between course_run_ids and program_uuids so they can be used
+        interchangeably. The two query parameters are left in for backwards compatability with edx-enterprise.
```

Comment:
Nit: there's only one query parameter, right?

Reply:
Both of the `contains_content_items`-related endpoints accept 2 query params (`course_run_ids` and `program_uuids`) for backwards compatibility with the existing endpoints in edx-enterprise. However, the endpoints in the enterprise-catalog service essentially [merges those 2 query params together](https://github.com/edx/enterprise-catalog/blob/9f6f27ae1689bd9db5d35057d2587e3e862088ea/enterprise_catalog/apps/api/v1/views.py#L106).

This comment and the one for the above `contains_content_items` method are basically just noting that while we're only sending the `course_run_ids` query parameter here, it's OK if the `content_ids` argument contains program ids since they get merged in together with the `course_run_ids` anyways.